buttons/main.scss

```diff
@@ -1,15 +1,8 @@
 @include nUiCriticalStart('buttons');
 
-$_n-ui-buttons-applied: false !default;
-$_n-ui-buttons-is-silent: nUiHas('buttons') == false or $_n-ui-buttons-applied == true;
-$o-buttons-is-silent: $_n-ui-buttons-is-silent;
+$o-buttons-is-silent: false !default;
```

Comment:
Had some... intense talks re. silent mode of late. The outcome of which was to let the o-component do its thing - I've added a note to the wiki too https://github.com/Financial-Times/n-ui/wiki/modules#sass